goas/cl/cl/src/partial_tx.rs

```diff
@@ -44,14 +46,30 @@ pub struct PartialTxWitness {
     pub outputs: Vec<OutputWitness>,
 }
 
-impl PartialTx {
-    pub fn from_witness(w: PartialTxWitness) -> Self {
-        Self {
-            inputs: Vec::from_iter(w.inputs.iter().map(InputWitness::commit)),
-            outputs: Vec::from_iter(w.outputs.iter().map(OutputWitness::commit)),
+impl PartialTxWitness {
+    pub fn commit(&self) -> PartialTx {
+        PartialTx {
+            inputs: Vec::from_iter(self.inputs.iter().map(InputWitness::commit)),
+            outputs: Vec::from_iter(self.outputs.iter().map(OutputWitness::commit)),
         }
     }
 
+    pub fn balance_delta(&self) -> i128 {
+        let in_sum: i128 = self.inputs.iter().map(|i| i.note.value as i128).sum();
```

Comment:
is this ignoring units?

Reply:
true, this is not used actually, I'll just get rid of it.